### DISCUSSION TITLE
ACME and obtaining certificate in Windows-installation

### DISCUSSION THREAD

**Discussion by larsfredrikbach** [drakkan/sftpgo#1227] [Q&A]
Hello,

I am trying to enable https for SFTPGo in a Windows Server Installation.
I have defined sftpgo\env.d\acme.env:
(real domain and email is left out for security-reason)
SFTPGO_ACME__DOMAINS="mydomain.no"
SFTPGO_ACME__EMAIL="email@email.com"
SFTPGO_ACME__HTTP01_CHALLENGE__WEBROOT=""

I get a private.key under sftpgo\certs\ but no certificate-file is obtained.

When running PS C:\Program Files\SFTPGo> .\sftpgo.exe acme run
2023-03-10T20:37:21.516 INF no domains configured, acme disabled
2023-03-10T20:37:21.517 ERR Cannot get certificates: acme is disabled

What could be wrong, and why is it not obtaining any certificate?
Should I define webroot?

**Comment by larsfredrikbach** [comment]
After browsing through sftpgo.log I found the following error:
{"level":"debug","time":"2023-03-10T21:25:24.167","sender":"acme","message":"configuring HTTP-01 challenge, port 80"}
{"level":"error","time":"2023-03-10T21:25:24.167","sender":"acme","message":"cannot renew certificates, your account is not registered"}
{"level":"error","time":"2023-03-10T21:25:24.167","sender":"acme","message":"unable to renew certificates: cannot renew certificates, your account is not registered"}

so my next question is, how do I register my account?
I have tried to run sftpgo acme run but that gives me error : no domains configures, acme disabled.

**Reply by drakkan** [reply]
On Windows the config dir should be `C:\ProgramData\SFTPGo`.  Execute `sftpgo acme run` from there or set the config dir like this

```
sftpgo acme run -c <path to the config dir>
``` 

You might also try the development version which allows you to get certificates from the WebUI. You can download a Windows setup from the actions page

https://github.com/drakkan/sftpgo/actions/runs/4386481655

This way you at least help testing the development version. Thank you

**Reply by larsfredrikbach** [reply]
Thank you very much. I got to obtain a certificate now.

I will try the development version.
